common/src/warp_runner/manager/commands/blink_commands.rs

```diff
@@ -5,6 +5,11 @@ use warp::{blink::AudioCodec, crypto::DID};
 
 use crate::warp_runner::Calling;
 
+type DeviceResult = (
```

Comment:
I recommend changing this to a struct with fields `available_devices` and `selected_device`. Eliminating the tuple would result in one less thing to memorize.